apps/meteor/client/views/admin/rooms/EditRoom.tsx

```diff
@@ -97,7 +97,7 @@ const EditRoom = ({ room, onChange, onDelete }: EditRoomProps) => {
 	const handleArchive = useArchiveRoom(room);
 
 	const handleUpdateRoomData = useEffectEvent(async ({ isDefault, roomName, favorite, ...formData }) => {
-		const data = getDirtyFields(formData, dirtyFields);
+		const data = getDirtyFields({ isDefault, roomName, favorite, ...formData }, dirtyFields);
```

Comment:
API doesn't accept `isDefault` field and that case is taken into account below, so this can cause errors when this field is update, I would suggest that we remove `isDefault` from here.